client/components/mma/accountoverview/ContributionUpdateAmountForm.tsx

```diff
@@ -161,7 +161,7 @@ export const ContributionUpdateAmountForm = (
 	const currentContributionOptions = (contributionAmountsLookup[
 		props.mainPlan.currencyISO
 	] || contributionAmountsLookup.international)[
-		props.mainPlan.interval as ContributionInterval
+		props.mainPlan.billingPeriod as ContributionInterval
```

Comment:
Similar to above comment, should ContributionInterval also change

Reply:
yep Ill amend it, the only thing I was thinking of leaving is the `ContributionUpdateAmount` component, as we use the word `amount` in manage.theguardian.com for contributions